### ISSUE TITLE
Optional<Any> to Any? causes conformance error

### ISSUE THREAD

**Issue by jnordberg** [nicklockwood/SwiftFormat#925]
SwiftFormat formats the following code:

```swift
switch value {
case Optional<Any>.none:
}
```

to:

```swift
switch value {
case Any?.none:
}
```

This causes the code to fail to compile with: `Protocol 'Any' as a type cannot conform to 'Equatable'`.

**Comment by nicklockwood** [comment]
@jnordberg thanks for reporting this. I'll see if I can work out why this causes a compiler error, but in the meantime you can disable the rule by adding `--disable typeSugar` to your .swiftformat config file or CLI arguments.

**Comment by nicklockwood** [comment]
@jnordberg fixed in 0.48.1
